internal/api/sms_provider/twilio.go

```diff
@@ -63,31 +63,43 @@ func NewTwilioProvider(config conf.TwilioProviderConfiguration) (SmsProvider, er
 	}, nil
 }
 
-func (t *TwilioProvider) SendMessage(phone string, message string, channel string) (string, error) {
+func (t *TwilioProvider) SendMessage(phone, message, channel, otp string) (string, error) {
 	switch channel {
 	case SMSProvider, WhatsappProvider:
-		return t.SendSms(phone, message, channel)
+		return t.SendSms(phone, message, channel, otp)
 	default:
 		return "", fmt.Errorf("channel type %q is not supported for Twilio", channel)
 	}
 }
 
 // Send an SMS containing the OTP with Twilio's API
-func (t *TwilioProvider) SendSms(phone, message, channel string) (string, error) {
+func (t *TwilioProvider) SendSms(phone, message, channel, otp string) (string, error) {
 	sender := t.Config.MessageServiceSid
 	receiver := "+" + phone
+	var body url.Values
 	if channel == WhatsappProvider {
 		receiver = channel + ":" + receiver
 		if isPhoneNumber.MatchString(formatPhoneNumber(sender)) {
 			sender = channel + ":" + sender
 		}
+		contentVariables := fmt.Sprintf(`{"1": "%s"}`, otp)
```

Comment:
can we add a comment as to why this needs to be set to ``{"1": "%s"}`? i'm guessing this is the format that Twilio's API requires